tests/cli/test_cli_regression_mol.py

```diff
@@ -141,6 +144,58 @@ def test_train_output_structure_cv_ensemble(monkeypatch, data_path, tmp_path):
     assert (tmp_path / "fold_2" / "train_smiles.csv").exists()
 
 
+def test_train_csv_splits(monkeypatch, data_dir, tmp_path):
+    input_path = str(data_dir / "regression" / "mol" / "mol_with_splits.csv")
+    args = [
+        "chemprop",
+        "train",
+        "-i",
+        input_path,
+        "--smiles-columns",
+        "smiles",
+        "--target-columns",
+        "lipo",
+        "--epochs",
+        "1",
+        "--num-workers",
+        "0",
+        "--save-dir",
+        str(tmp_path),
+    ]
+
+    with monkeypatch.context() as m:
+        m.setattr("sys.argv", args)
+        main()
+
+
+def test_train_splits_file(monkeypatch, data_path, tmp_path):
+    input_path, *_ = data_path
+    splits_file = str(tmp_path / "splits.json")
+    splits = [{"train": [1, 2], "val": "3-5", "test": "6,7"}]
```

Comment:
I think it would be better to extend the testing to include multiple components and cross-validation splits.

Reply:
Maybe good but not a reason to block IMO